dGame/dComponents/InventoryComponent.cpp

```diff
@@ -79,28 +79,29 @@ InventoryComponent::InventoryComponent(Entity* parent, tinyxml2::XMLDocument* do
 
 		subItems.erase(std::remove_if(subItems.begin(), subItems.end(), ::isspace), subItems.end());
 
-		if (!subItems.empty()) {
-			const auto subItemsSplit = GeneralUtils::SplitString(subItems, ',');
+		if (subItems.empty()) return;
```

Comment:
should be continue, not return
If I'm following correctly

Reply:
Yes